spec/models/user_spec.rb

```diff
@@ -2,4 +2,24 @@
 
 RSpec.describe User, type: :model do
 
-end
\ No newline at end of file
+    describe '#follow' do
+      it "allows a user to follow another" do
+        user1 = FactoryGirl.create(:user)
+        user2 = FactoryGirl.create(:user, email: "newuser2@sb.com")
```

Comment:
These should be in `let` or `subject` blocks. In most cases, you don't want to do an assignment inside of an `it` block.